modules/sdk-coin-ada/src/transaction.ts

```diff
@@ -1,17 +1,185 @@
-import { BaseTransaction, BaseKey, NotImplementedError } from '@bitgo/sdk-core';
+import {
+  BaseKey,
+  BaseTransaction,
+  Entry,
+  InvalidTransactionError,
+  TransactionType,
+  TransactionExplanation,
+} from '@bitgo/sdk-core';
+import * as CardanoWasm from '@emurgo/cardano-serialization-lib-nodejs';
```

Comment:
How will we be handling using the sdk in the browser given that this seems to be a nodejs package?

Reply:
We wrap it within @cardano-serialization-lib-browser which is compatible :) The polyfill code for that is in bitgo.config.js